apps/11gist-blog/script.js

```diff
@@ -1,5 +1,57 @@
 'use strict';
 
+var gistDescription = {
+  success: function(gists) {
+    gists.forEach(function(gist) {
+      var gistDesc = gist.description;
+      // .substring property to find "#post" in the gistDesc
+      var postStr = gistDesc.substring(0, 5);
+      if (postStr === "#post") {
+        // .replace property to remove "#post"
+        gistDesc = gistDesc.replace("#post", "");
+        var links = 
+        ('<li>' + '<a href="#" data-url="' + gist.url + '">' + gistDesc + '</a>' + '</li>');
+        $('#posts').append(links);
+      };
+    });
+  }
+};
+
+// callback to get the content of the gist that was clicked
+var gistContent = {
+  success: function(gist) {
+    var content = gist.files['post.md']['content'];
+    var commentURL = gist["comments_url"];
+    var str = 
+    ('<div id="post">' +
+    '<p>' + marked(content) + '</p>' + '</div>')
+    $('#post').replaceWith(str);
+    var getComments = {
```

Comment:
Please nest your `getComments` callback outside of the `gistContent` callback like you handle `gistContent` on line 55. 

This will make your logic much easier to read.